R/cache-disk.R

```diff
@@ -193,6 +198,9 @@
 #' @export
 cache_disk <- function(
   dir = NULL,
+  read_fn = NULL,
+  write_fn = NULL,
+  extension = ".rds",
```

Comment:
I'm pretty sure that adding these args here will break a nontrivial amount of existing code. Can you put them later? I think that they should at least be after `evict` or `destroy_on_finalize`.

This isn't the first time I've wished that I had used `...` as a placeholder to force people to use named args.